assembl/static2/js/app/components/common/media.jsx

```diff
@@ -1,20 +1,55 @@
-import React from 'react';
+// @flow
+import * as React from 'react';
 import { I18n } from 'react-redux-i18n';
 import { Grid, Col, Button, Image, ResponsiveEmbed } from 'react-bootstrap';
 
 import { displayModal } from '../../utils/utilityManager';
 
-class Media extends React.Component {
-  static isValidDescription = description => !!(description && description !== '<p></p>');
+type File = {
+  externalUrl: string
+};
+type Props = {
+  title: string,
+  descriptionTop: string,
+  descriptionBottom: string,
+  descriptionSide: string,
+  htmlCode: string,
+  mediaFile: File,
+  noTitle: boolean
+};
 
-  static Title = ({ value }) => (
+type TitleProps = {
+  value: ?string // eslint-disable-line
```

Comment:
why eslint-disable-line ? even for the line below ?

Reply:
it's an oversight! :+1: